tests/testthat/test-create-plots-from-excel.R

```diff
@@ -724,6 +724,47 @@ test_that("It exports plot grids as defined in sheet `exportConfiguration`", {
         projectConfiguration = projectConfigurationLocal,
         stopIfNotFound = TRUE
       )
+      # Get the name of the created folder (if successfull)
+      # Using tail in case there are multiple folders, so we just need the latest one
+      subFolderName <- tail(list.files(file.path(tempDir, "Figures")), n = 1)
```

Comment:
This looks ugly but I did not find another way to test. The path is created based on the current time stamp.